data/common.yaml

```diff
@@ -0,0 +1 @@
+python::default_system_version: "3.11"
```

Comment:
For the common file, the choice of `3.11` was arbitrary. We really only want a default for systems missing in Hiera data (e.g. operating systems not listed as supported).  For example the test suite mocks Debian 6, and we do not want to add support for this in Hiera :upside_down_face: